analyzer-java/src/main/java/com/infosupport/ldoc/analyzerj/AnalysisVisitor.java

```diff
@@ -71,12 +71,16 @@ public class AnalysisVisitor extends GenericListVisitorAdapter<Description, Anal
 
   private final SymbolResolver resolver;
 
-  /** Construct an AnalysisVisitor that uses the given SymbolResolver to find the names of types. */
+  /**
+   * Construct an AnalysisVisitor that uses the given SymbolResolver to find the names of types.
+   */
```

Comment:
Here and below: do these really need to change? The single-line Javadoc is explicitly allowed per the [Java style guide](https://google.github.io/styleguide/javaguide.html#s7.1.1-javadoc-multi-line).

Reply:
I assume this all happened after i reformatted the code, wierd that it formatted them like that i didnt notice.